x-pack/plugins/fleet/server/services/spaces/agent_policy.ts

```diff
@@ -23,9 +23,10 @@ import { ENROLLMENT_API_KEYS_INDEX } from '../../constants';
 import { packagePolicyService } from '../package_policy';
 import { FleetError, HostedAgentPolicyRestrictionRelatedError } from '../../errors';
 
-import { isSpaceAwarenessEnabled } from './helpers';
 import type { UninstallTokenSOAttributes } from '../security/uninstall_token_service';
 
+import { isSpaceAwarenessEnabled } from './helpers';
+
```

Comment:
@elastic/fleet  team, Not sure why but the linter is making this change after merging main 🤷 . May be a new rule?